File: CS/DxWebApp/Models/ThemesModel.cs
using System;

namespace DxWebApp.Models {
    public enum CommonThemes {
        Moderno,
        Material,
        MaterialCompact
    }

    public class ThemesModel {
        public String Name { get; set; }
        public CommonThemes Theme { get; set; }
        public string ThemeColor { get; set; }
        public string Font { get; set; }
    }
}

File: VB/DxWebApp/Models/ThemesModel.vb
Imports System

Namespace DxWebApp.Models
    Public Enum CommonThemes
        Moderno
        Material
        iOS
    End Enum

    Public Class ThemesModel
        Public Property Name() As String
        Public Property Theme() As CommonThemes
        Public Property ThemeColor() As String
        Public Property Font() As String
    End Class
End Namespace